docs-src/go/auto-restart-worker-tip.md

```diff
@@ -0,0 +1,20 @@
+---
+id: auto-restart-worker-tip
+title: Automatically restart Worker Process during development
```

Comment:
Curious why we don't talk about automatically restarting Worker Process in TypeScript? We can use nodemon. Not sure if any of the other SDK have similar methods.